index.ts

```diff
@@ -63,13 +66,15 @@ async function main() {
     for (const projectIssue of projectIssues) {
       // if issue exists in devpool
       const devpoolIssue = getIssueByLabel(devpoolIssues, `id: ${projectIssue.node_id}`);
```

Comment:
Even in a forked repo this code will fetch devpool issues which we don't really need

Reply:
Could you elaborate please? I don't think i understand

Reply:
My mistake, I actually thought that (as described in the original issue) this repo should be forked and run against `ubiquity/devpool-rfc` hence I [asked](https://github.com/ubiquity/devpool-directory/pull/1212#discussion_r1658756763) for using env variables and [this](https://github.com/ubiquity/devpool-directory/pull/1212#discussion_r1658775169) change.